buku

```diff
@@ -5850,6 +5850,10 @@ POSITIONAL ARGUMENTS:
         if args.colorstr is None and colorstr_env is not None:
             args.nc = True
 
+    # Handle NO_COLOR as well:
+    if os.environment.get('NO_COLOR') is not None:
+        args.nc = True
+
```

Comment:
Have you actually tried to _run_ this code? The lint fails because there's no `environment` in the `os` module :sweat_smile:
(…try [`getenv()`](https://docs.python.org/3/library/os.html#os.getenv) instead)